api/v1/sriovnetworknodestate_types.go

```diff
@@ -90,6 +90,7 @@ type SriovNetworkNodeStateStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="Sync Status",type=string,JSONPath=`.status.syncStatus`
```

Comment:
Please, add something like `// +kubebuilder:printcolumn:name="Age",type=string,JSONPath=`.metadata.creationTimestamp``  (I didn't test this change with sriov operator so this line could be inacurate) to show resource age as we've got it now

Reply:
Sure!